notebooks/vertex_genai/solutions/gen_ai_evaluation_service.ipynb

```diff
@@ -0,0 +1,812 @@
+{
```

Comment:
<p>Add links to MetricX and COMET.</p><p> </p><p><a href="https://github.com/google-research/metricx" target="_blank">https://github.com/google-research/metricx</a></p><p><a href="https://huggingface.co/Unbabel/wmt22-comet-da" target="_blank">https://huggingface.co/Unbabel/wmt22-comet-da</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Added! Thanks! </p>